apps/src/templates/instructions/AniGifPreview.jsx

```diff
@@ -8,20 +8,34 @@ import {openDialog} from '../../redux/instructionsDialog';
 class ImagePreviewUnwrapped extends React.Component {
   static propTypes = {
     url: PropTypes.string.isRequired,
+    alt: PropTypes.string,
     showInstructionsDialog: PropTypes.func.isRequired,
     noVisualization: PropTypes.bool.isRequired,
   };
 
+  handleKeyDown = e => {
+    if (e.key === 'Enter' || e.key === ' ') {
+      e.preventDefault();
+      this.props.showInstructionsDialog();
+    }
+  };
+
   render() {
+    const {url, alt, showInstructionsDialog, noVisualization} = this.props;
     return (
       <div id="ani-gif-preview-wrapper" style={styles.wrapper}>
         <div
           id="ani-gif-preview"
+          role="button"
+          tabIndex={0}
+          aria-label={alt || undefined}
```

Comment:
Would it be possible for us to follow the WCAG guidelines for images and have empty ('') alt text if it is not given?